lib/themes/dosomething/paraneue_dosomething/js/images/ImageUpload.js

```diff
@@ -75,6 +75,7 @@ define(function(require) {
       }
 
       if (!_this.validFileSize(files)) {
+        ga('send', 'event', 'Reportback', 'image file too large');
```

Comment:
Just to be safe, I'd recommend wrapping this in a quick check that the `ga` object exists:

```js
if (typeof ga !== 'undefined' && ga !== null) {
  // your analytics event
}
```